data/json/monsters/zed_explosive.json

```diff
@@ -59,7 +97,7 @@
     "death_function": { "message": "The %s explodes!", "effect": { "id": "death_gas", "hit_self": true }, "corpse_type": "NO_CORPSE" },
     "fungalize_into": "mon_gasbag_dusted",
     "armor": { "electric": 1 },
-    "delete": { "flags": [ "BILE_BLOOD", "BASHES", "GROUP_BASH" ], "special_attacks": [ "BOOMER" ] }
+    "delete": { "flags": [ "BILE_BLOOD", "BASHES", "GROUP_BASH" ], "special_attacks": [  ] }
```

Comment:
If this works it's pretty gross, shouldn't it just be "special_attacks": [  ] not inside a delete?